Autopilot/Inc/IMU.hpp

```diff
@@ -1,71 +0,0 @@
-/**
```

Comment:
we delete this file ? Is it affecting gps somehow ?

Reply:
Oh, when I made the gps branch, it included the changes I made to the IMU files. So basically It just had outdated changes in the IMU PR. 

I deleted the files so we don't merge them by accident.